web_ui/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
I'm thinking we have it so you open vscode in the root of the repo

thoughts?

Reply:
I'm doubtful that will work with two python projects, no?